src/sse.ts

```diff
@@ -75,6 +77,12 @@ export class AblySSEChannel {
   }
 
   private async refreshToken() {
+    const cached = getAuthToken(this.userId);
```

Comment:
This can throw, but we have no try/catch

Reply:
can't throw anymore